server/src/Dao/RevisionDao.php

```diff
@@ -10,16 +10,17 @@ class RevisionDao extends AbstractDao {
 	 * Fetch user interaction in commonly edited pages
 	 *
 	 * @param int[] $users
-	 * @param null $startDate
-	 * @param null $endDate
+	 * @param \DateTime|null $startDate
+	 * @param \DateTime|null $endDate
 	 * @param int $limit
 	 * @param null $continue
 	 * @return array
 	 */
 	public function getUserRevisionsInCommonPages(
-		array $users, $startDate = null, $endDate = null, $limit = 50, $continue = null
+		array $users, \DateTime $startDate = null, \DateTime $endDate = null,
```

Comment:
nit-pick typehint the `\DateTimeInterace`
https://secure.php.net/manual/en/class.datetimeinterface.php